tests/libres_tests/res/enkf/test_analysis_iter_config.py

```diff
@@ -23,6 +23,7 @@
 class AnalysisIterConfigTest(ResTest):
     def test_set_analysis_iter_config(self):
         c = AnalysisIterConfig()
+        self.assertTrue(str(c).startswith("AnalysisIterConfig"))
```

Comment:
Maybe test repr directly? str(obj) falls back to __repr__ when __str__ is not implemented
```suggestion
        self.assertTrue(repr(c).startswith("AnalysisIterConfig"))
```

Reply:
👍